tests/layers_views/views/DetailView.spec.js

```diff
@@ -14,11 +14,25 @@ generateViewTests(DetailView, detailViewArguments);
 test(`DetailView layer type and props check`, t => {
   const view = new DetailView(detailViewArguments);
   const loader = { type: 'loads', isPyramid: true };
-  const layers = view.getLayers({ props: { loader } });
+  const layers = view.getLayers({
+    props: {
+      loader: { ...loader, PhysicalSizeXUnit: 'cm', PhysicalSizeX: 0.5 }
```

Comment:
why not just define these on the loader above?

Reply:
The base test just checks for id-passing.  This one checks that proper layers are rendered, for which this is needed and in the former case it is not.  The OverviewView, for example, doesn't technically need this information.